src/components/ToggleSwitch/docs/index.story.tsx

```diff
@@ -43,7 +44,9 @@ export default {
       tab({
         title: 'Usage',
         sections: [
-          description(Readme),
+          description(
+            '`ToggleSwitch` is a component allowing to render a sort of button that toggles between on and off states.',
```

Comment:
I think this can be:
```suggestion
            '`ToggleSwitch` is a component allowing to render a button with on and off states.',
```